test/unit/npm-dependncies.spec.js

```diff
@@ -0,0 +1,58 @@
+const npmDependencies = require("../../src/dynamicTypes/npmDependencies");
+
+describe("npm dependencies", () => {
+  describe("should load types for npm modules", () => {
+    it("get default type when request raw", () => {
+      const rawNpmType = npmDependencies.getRaw("react");
+      expect(rawNpmType).toMatchSnapshot();
+    });
+
+    it("get default type when request files and the namespaces is not @wix", async () => {
+      const npmDtsFiles = await npmDependencies.getFiles({
+        react: "16.8.6",
+        "react-dom": "16.8.6"
+      });
+      expect(npmDtsFiles).toMatchSnapshot();
+    });
+
+    it("get proper type for @wix scoped npm module", async () => {
+      global.fetch = jest.fn(() =>
```

Comment:
I suggest to make it more explicit.
* spy only on the specific request
* extract the "boilerplate" to a function
something like
`givenDependencies('some-package-name', "some types")`

Reply:
I'll refactor that 🙏